modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminDisplayContext.java

```diff
@@ -802,15 +805,24 @@ private Hits _getHits(SearchContainer<RepositoryEntry> searchContainer)
 		SearchContext searchContext = SearchContextFactory.getInstance(
 			_httpServletRequest);
 
-		searchContext.setAttribute("paginationType", "regular");
-
 		long searchRepositoryId = ParamUtil.getLong(
 			_httpServletRequest, "searchRepositoryId");
 
 		if (searchRepositoryId == 0) {
 			searchRepositoryId = _themeDisplay.getScopeGroupId();
 		}
```

Comment:
Can you use `_themeDisplay.getScopeGroupId()` as the default value and remove the conditional?
```
long searchRepositoryId = ParamUtil.getLong(
	_httpServletRequest, "searchRepositoryId", _themeDisplay.getScopeGroupId());
```

Reply:
@adolfopa, @marcogalluzzi
I updated code as you guided. However, could you help to double check the 
```
GroupPermissionUtil.contains(
				_themeDisplay.getPermissionChecker(), group, ActionKeys.VIEW)
```
Because if we add this condition, it no longer works as expectation when searching without login following steps in the https://issues.liferay.com/browse/LPS-172887. It shows different results

Reply:
If we still keep this condition, why without searching, Anonymous can see the resource, but can't see when doing the searching action?

Reply:
When searching in a regular group that is connected to the asset library, the user should have the ASSET_LIBRARY_CONNECTED_SITE_MEMBER role assigned to him automatically (see DepotRoleContributor class). By default that role should give view permissions (both in the search and individual access). So, that permission check should be safe to remove; when checking the VIEW permissions over each returned entry (done by the search infra.), those that the user should not see will be removed automatically.

Reply:
Thank for your explanation, the current code already is updated based on your suggestion. Please help me review and move on the next step